txstratum/protocol.py

```diff
@@ -151,8 +158,17 @@ def handle_result(self, result: Any, msgid: JSONRPCId) -> None:
         message = self.messages_in_transit.pop(msgid)
 
         if message.method == "client.get_version":
-            self.log.info("Miner version: {}".format(result), miner_id=self.miner_id)
+            if self.miner_version is None:
+                # Logs only the first time, otherwise this log would appear a lot because
+                # we use this method to estimate the RTT
+                self.log.info(
+                    "Miner version: {}".format(result), miner_id=self.miner_id
+                )
             self.miner_version = result
+            now = txstratum.time.time()
+            current_rtt = now - message.created_at
+            alpha = 0.7
```

Comment:
Maybe move to a constant at the class level.

Reply:
Done https://github.com/HathorNetwork/tx-mining-service/pull/114/commits/413e9e2f80260e45b8f23f3b876d6f975599ce69

Reply:
Thanks!